src/constants/tokenLists/avax.mainnet.tokenlist.ts

```diff
@@ -51,6 +51,13 @@ export const AVAX_MAINNET_TOKEN_LIST = {
       symbol: 'APEIN',
       name: 'Ape In',
       decimals: 18
+    },
+    {
+      chainId: 43114,
+      address: '0x961c8c0b1aad0c0b10a51fef6a867e3091bcef17',
```

Comment:
```suggestion
      address: '0x961C8c0B1aaD0c0b10a51FeF6a867E3091BCef17',
```

can you please use the checksummed version: 0x961C8c0B1aaD0c0b10a51FeF6a867E3091BCef17 ?

Other than that, this PR is ok